net/SslSocket.hpp

```diff
@@ -414,11 +414,19 @@ class SslStreamSocket final : public StreamSocket
                                           << bioErrStr);
 #if OPENSSL_VERSION_NUMBER > 0x10100000L
                 else if (sslError == SSL_ERROR_WANT_ASYNC)
-                    LOG_TRC("SSL error (" << context << "): WANT_ASYNC (" << sslError << ") "
+                {
+                    LOG_WRN("SSL error (" << context << "): WANT_ASYNC (" << sslError << ") "
                                           << bioErrStr);
+                    LOG_ASSERT_MSG(sslError != SSL_ERROR_WANT_ASYNC,
```

Comment:
Not a very obvious way of firing an assertion in this path ;-) != the value we know it is.

Reply:
Changed in the follow-up.